numpy/core/shape_base.py

```diff
@@ -177,6 +160,81 @@ def atleast_3d(*arys):
         return res
 
 
+def atleast_nd(ary, n, pos='start'):
+    """
+    View input as array with at least `n` dimensions.
+
+    Parameters
+    ----------
+    ary : array_like
+        The input array. Non-array inputs are converted to arrays.
+        Arrays that already have `n` or more dimensions are preserved.
+    n : scalar
+        The minimum number of dimensions required for each of the input
+        arrays.
```

Comment:
there's only ever 1 array, for each of the input arrays doesn't seem to make sense